hx_lti_initializer/utils.py

```diff
@@ -128,6 +128,8 @@ def save_session(request, **kwargs):
         "roles": ["hx_roles", []],
         "is_staff": ["is_staff", False],
         "is_instructor": ["is_instructor", False],
+        "is_graded": ['is_graded', False],
```

Comment:
Looking at the code as it's written, is is only set if `save_session()` is called with `is_graded` as a keyword argument. That's probably not the most intuitive behavior though, because like you said, I would have expected the default to be set even if `save_session()` wasn't called. Maybe something to look into as part of this change, or separately, perhaps to limit the scope of this PR to just the grading feature.